spec.md

```diff
@@ -460,6 +461,21 @@ Analog<32> ; 32-bit analog type
 Analog     ; analog type with inferred width
 ```
 
+### String Type
+
+The string type specifies that a value will be a string.  String types do not
+specify a width.  String types may be a component of an aggregate.  String types
+follow FIRRTL connection semantics and last-connect semantics.  The encoding of
+a string is implementation defined.
+
+``` firrtl
+parameter foo = "hello" ; a parameter with a string type
```

Comment:
is this the first time `parameter` is in the spec?

Reply:
No. It's used in the `extmodule` definition. I'm not (yet) introducing a first-class parameter. (That is step 2 after getting a first-class string type. 😅)

Perhaps this example was confusing as I'm lumping a bunch of different statements together and the `parameter foo = "hello"` wouldn't appear with the others in the current spec version.